lib/stitches/api_generator.rb

```diff
@@ -32,10 +31,6 @@ def bootstrap_api
     # as well as for your client to be able to validate this as well.
   end
 end
-api_docs = Rack::Auth::Basic.new(Apitome::Engine ) do |_,password|
-  password == ENV['HTTP_AUTH_PASSWORD']
-end
-mount api_docs, at: "docs"
```

Comment:
I'm suggesting we remove it altogether, however it should also be possible to keep this in the "suggested" configuration but removed from the gem's runtime dependency so we have the flexibility to remove it if we don't want to server the API documentation in production.